kaggle_environments/envs/halite/helpers.py

```diff
@@ -16,138 +16,12 @@
 from enum import Enum, auto
 from functools import wraps
 from typing import *
-import operator
+from kaggle_environments.helpers import *
 import sys
 
 
-# region Helper Classes and Methods
```

Comment:
These have all moved up two levels to become a shared `kaggle_environments.helpers` class that provides useful cross-environment utilities.